packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -192,6 +216,59 @@ export const outOfOfficeCreate = async ({ ctx, input }: TBookingRedirect) => {
     }
   }
 
+  // Send webhook to notify other services
+  const subscriberOptions: GetSubscriberOptions = {
+    userId: ctx.user.id,
+    orgId: ctx.user.organizationId,
+    triggerEvent: WebhookTriggerEvents.OOO_CREATED,
+  };
+
+  const subscribers = await getWebhooks(subscriberOptions);
+  await Promise.all(
+    subscribers.map(async (subscriber) => {
+      sendPayloadNoBooking(
+        subscriber.secret,
+        WebhookTriggerEvents.OOO_CREATED,
+        dayjs().toISOString(),
+        {
+          appId: subscriber.appId,
+          subscriberUrl: subscriber.subscriberUrl,
+          payloadTemplate: subscriber.payloadTemplate,
+        },
+        {
+          metadata: {
```

Comment:
```suggestion
```

I don't think metadata makes sense, let's improve the type for `data` in `sendPayloadNoBooking`

Reply:
After removing sendPayloadNoBooking, I still believe that ooo entries data shouldn't go right there with booking webhook data. So metadata makes sense still here.
Unless you want me to redo the type for metadata and allow booking & new fields there.